src/irsForms/ScheduleEIC.test.ts

```diff
@@ -0,0 +1,52 @@
+import { waitFor } from '@testing-library/react'
+import { FilingStatus, PersonRole, TaxesState } from '../redux/data'
+import ScheduleEIC from './ScheduleEIC'
+import F1040 from './F1040'
+
+afterEach(async () => {
+  await waitFor(() => localStorage.clear())
+  jest.resetAllMocks()
+})
+
+jest.mock('redux-persist', () => {
+  const real = jest.requireActual('redux-persist')
+  return {
+    ...real,
+    persistReducer: jest.fn().mockImplementation((_, reducers) => reducers)
+  }
+})
+
+beforeAll(async () => {
+  jest.spyOn(console, 'log').mockImplementation(() => {})
+  jest.spyOn(console, 'warn').mockImplementation(() => {})
+  jest.spyOn(console, 'info').mockImplementation(() => {})
+  jest.spyOn(console, 'debug').mockImplementation(() => {})
```

Comment:
from looking at the scheduleEIC file, it looks like only console.warn needs to be mocked with a noop to silence output

Reply:
Fixed this